std_msgs/Int16 obj_id
std_msgs/Int16 bin_id
---
